server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -772,15 +761,8 @@ public void run() {
     }
   }
 
-  public String getClientAddressString() {
-    if (clientAddress == null) {
-      return null;
-    }
-    return clientAddress.getHost() + ":" + clientAddress.getPort();
-  }
-
   public TServerInstance getTabletSession() {
-    String address = getClientAddressString();
+    String address = getAdvertiseAddress().toString();
     if (address == null) {
```

Comment:
address cannot be null here. Probably want .equals("null") or check if getAdvertiseAddress() == null at the start of the method (I prefer the latter)

Reply:
Changed this in b79033a